server/src/test/java/com/objectcomputing/checkins/services/memberprofile/MemberProfileControllerTest.java

```diff
@@ -22,10 +22,7 @@
 import java.net.URLEncoder;
 import java.nio.charset.StandardCharsets;
 import java.time.LocalDate;
-import java.util.Map;
-import java.util.Objects;
-import java.util.Set;
-import java.util.UUID;
+import java.util.*;
 
```

Comment:
On other projects I have been dinged for start imports, just something to keep in mind.